tests/PHPStan/Rules/Arrays/NonexistentOffsetInArrayDimFetchRuleTest.php

```diff
@@ -127,6 +131,10 @@ public function testRule(): void
 				'Offset int does not exist on array<string, string>.',
 				312,
 			],
+			[
+				'Offset int|null might not exist on array<string, string>.',
```

Comment:
Well this is totally wrong, `null` gets cast to `''` in an array key and that can exist on `array<string, string>.`. Not sure where to do the fix.